prow/config.yaml

```diff
@@ -50,15 +50,31 @@ github_reporter:
   - presubmit
   - postsubmit
 
-blocked_branches: &blocked_branches
-  release-*: &blocking_merge
+blocked_branches: &blocked_branches_base
+  master:
+    protect: true
+  release-0.1: &blocking_merge
     required_status_checks:
       contexts:
       - "merges-blocked-needs-admin"
     protect: true
+  release-0.2: *blocking_merge
+  release-0.3: *blocking_merge
+  release-0.4: *blocking_merge
+  release-0.5: *blocking_merge
+  release-0.6: *blocking_merge
+  release-0.7: *blocking_merge
+  release-0.8: *blocking_merge
+  release-1.0: *blocking_merge
+
+blocked_branches: &blocked_branches
```

Comment:
Why are these branches separate from the ones above? Does this have a semantic meaning?